### PR TITLE
data:sebashmh

### PR DESCRIPTION
**Pull request by sebashmh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5386"></a>

### CODE REVIEW

**Review by SurajPratap10 — CHANGES_REQUESTED** [review]
remove the `avatar` part as we don't need that now...

**Comment by sebashmh** [comment]
> remove the `avatar` part as we don't need that now...

@SurajPratap10 The property `avatar` has been removed.

**Review by SurajPratap10 — APPROVED** [review]
LGTM!  Thanks for contributing..

File: data/sebashmh.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        {
+      "name": "GitHub",
+      "url": "https://github.com/sebashmh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sebastianhenaomadrigal/",
+      "icon": "FaLinkedin"
+    },
```
Comment (MahirMahdi) [inline data/sebashmh.json:37]: Hey, take out the comma after the LinkedIn object. That's what made the JSON check fail.

**Review by MahirMahdi — CHANGES_REQUESTED** [review]

File: data/sebashmh.json (hunk 1 above)
Comment (sebashmh) [reply data/sebashmh.json]: The comma was removed. @MahirMahdi thanks for the feedback.

File: data/sebashmh.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    "Docker",
+    "Node.js",
+    "AWS"
+  ],
+    "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/madriantor" },
+    { "icon": "FaGithub", "url": "https://github.com/sebashmh" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/sebastianhenaomadrigal/" },
+    { "icon": "FaInstagram", "url": "https://vero.co/sebasmadrigal" },
+    { "icon": "FaLink", "url": "https://sebas.io" },
```
Comment (kumarsonsoff3) [inline data/sebashmh.json:20]: There is an extra comma leading json check fail. 

```suggestion
    { "icon": "FaLink", "url": "https://sebas.io" }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/sebashmh.json (hunk 2 above)
Comment (sebashmh) [reply data/sebashmh.json]: Comma removed, thanks.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**